pkg/tsdb/loki/loki.go

```diff
@@ -70,8 +71,9 @@ type datasourceInfo struct {
 
 type QueryJSONModel struct {
 	dataquery.LokiDataQuery
-	Direction           *string `json:"direction,omitempty"`
-	SupportingQueryType *string `json:"supportingQueryType"`
+	Direction           *string              `json:"direction,omitempty"`
+	SupportingQueryType *string              `json:"supportingQueryType"`
+	Scopes              []models.ScopeFilter `json:"scopes"`
```

Comment:
I decided to add these fields to this struct instead of `dataquery.LokiDataQuery` since we are not gonna use theme in the future.